packages/ui/src/viewer-table/InteractionPanel/CollapseAllButton.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react';
+import Button from '../../common/Button';
+import { useThemeContext } from '../../theme/ThemeContext';
+
+export interface CollapseAllButtonProps {
+	onClick: () => void;
+}
+
+const CollapseAllButton = ({ onClick }: CollapseAllButtonProps) => {
+	const theme = useThemeContext();
+	const { Collapse } = theme.icons;
+
+	return (
+		<Button leftIcon={<Collapse />} onClick={onClick}>
+			Collapse All
+		</Button>
+	);
+};
+
+export default CollapseAllButton;
```

Comment:
pop a copyright up top 
This is looking really good. Super clean. Easily readable. exported types. Specialized button building off a more generic button component. very nice